packages/gatsby/src/bootstrap/index.ts

```diff
@@ -46,6 +47,11 @@ export async function bootstrap(
     ...(await initialize(bootstrapContext)),
   }
 
+  const workerPool = context.workerPool
+  const directory = slash(context.store.getState().program.directory)
+
+  workerPool.all.loadConfigAndPlugins({ siteDirectory: directory })
```

Comment:
Do we want to wrap those two `.all` calls with the env var?

Reply:
Yes! Was about to mention this, as there is no point in loading config and buildign schema in workers unless we run queries there too